StantUiIosLibrary/Classes/Tables/ConstructionTableViewWithCollapsedSearchBar/ConstructionTableViewWithCollapsedSearchBar.swift

```diff
@@ -79,16 +74,26 @@ public class ConstructionTableViewWithCollapsedSearchBar: UIView {
                                           animationDelegate: self,
                                           selectCellDelegate: tableViewDelegate)
         tableView?.backgroundColor = UIColor.clear
+        guard let tableView = tableView else { return }
```

Comment:
I believe in this case you can use `if let` instead of `guard let`. The code is executing only one line after the `guard` without any more logic for it.

Reply:
Sure :)